terraform/provider/provider.go

```diff
@@ -171,6 +199,15 @@ func (p *Provider) Configure(ctx context.Context, req tfsdk.ConfigureProviderReq
 		creds = append(creds, cred)
 	}
 
+	if certBase64 != "" && keyBase64 != "" {
```

Comment:
What if we want to get the cert+key from Base64 env vars but the CA from a file?

I'm not sure this is a requirement but seems to be reasonable

Reply:
I am mostly following the logic applied for the path based approach, where we only check cert and key. Here my assumption is that if we don't have anything passed  for `rootCaBase64` it will end up being decoded to empty `[]byte{}` and optional - like loglines are suggesting `Using auth with certificate, private key and (optionally) CA`.

Reply:
Sounds good :+1: